core-bundle/src/EventListener/DoctrineSchemaListener.php

```diff
@@ -41,53 +37,4 @@ public function postGenerateSchema(GenerateSchemaEventArgs $event): void
     {
         $this->provider->appendToSchema($event->getSchema());
     }
-
-    /**
-     * Adds the index length on MySQL platforms for Doctrine DBAL <2.9.
-     */
-    public function onSchemaIndexDefinition(SchemaIndexDefinitionEventArgs $event): void
```

Comment:
This got removed because we now require `doctrine/dbal` in version `2.10`?

Reply:
Exactly.